tests/test_message.py

```diff
@@ -271,6 +271,16 @@ class Color(proto.Enum):
     assert new_s == s
 
 
+def test_serialize_to_dict_float_precision():
+    class Squid(proto.Message):
+        mass_kg = proto.Field(proto.FLOAT, number=1)
+
+    s = Squid(mass_kg=3.14159265)
+
+    s_dict = Squid.to_dict(s, float_precision=3)
+    assert s_dict["mass_kg"] == 3.14
```

Comment:
I would have expected this to be `3.141` or `3.142` since `float_precision=3`. If you agree, can you file a bug?

Reply:
Ok, I filed https://github.com/googleapis/proto-plus-python/issues/390

Should it block this PR?